File: mcp/tools/serena/test/resources/repos/fsharp/test_repo/Models/Person.fs
namespace Models

/// Person record type
type Person = {
    Name: string
    Age: int
    Email: string option
}

module PersonModule =
    /// Create a new person
    let createPerson name age email =
        { Name = name; Age = age; Email = email }
    
    /// Check if person is an adult
    let isAdult person =
        person.Age >= 18
    
    /// Get display name
    let getDisplayName person =
        match person.Email with
        | Some email -> $"{person.Name} ({email})"
        | None -> person.Name
    
    /// Update person age
    let updateAge newAge person =
        { person with Age = newAge }

/// Address type
type Address = {
    Street: string
    City: string
    ZipCode: string
    Country: string
}

/// Employee type that extends Person concept
type Employee = {
    Person: Person
    EmployeeId: int
    Department: string
    Salary: decimal
    Address: Address option
}

File: mcp/tools/serena/test/resources/repos/csharp/test_repo/Models/Person.cs
using TestProject;

namespace TestProject.Models
{
    public class Person
    {
        public string Name { get; set; }
        public int Age { get; set; }
        public string Email { get; set; }
        
        public Person(string name, int age, string email)
        {
            Name = name;
            Age = age;
            Email = email;
        }
        
        public override string ToString()
        {
            return $"{Name} ({Age}) - {Email}";
        }
        
        public bool IsAdult()
        {
            return Age >= 18;
        }
        
        public int CalculateYearsUntilRetirement()
        {
            var calculator = new Calculator();
            return calculator.Subtract(65, Age);
        }
    }
}